pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -1015,6 +1016,20 @@ type Address struct {
 
 //END Video -------------------
 
+//BEGIN VSOCK -------------------
+
+type VSOCK struct {
+	Model string `xml:"model,attr,omitempty"`
+	CID   CID    `xml:"cid"`
+}
+
+type CID struct {
+	Auto    string `xml:"auto,attr"`
+	Address string `xml:"address,attr,omitempty"`
```

Comment:
I think you can make this a `uint32` type directly.

Reply:
Done